_saas-integrations/workday/v1/workday-raas-v1.md

```diff
@@ -0,0 +1,133 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Workday RaaS
+permalink: /integrations/saas/workday-raas
+keywords: workday-raas, integration, schema, etl workday-raas, workday-raas etl, workday-raas schema
+layout: singer
+# input: false
+
+key: "workday-raas-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "workday-raas"
+display_name: "Workday RaaS"
+
+singer: true
+status-url: ""
+
+tap-name: "workday-raas"
+repo-url: https://github.com/singer-io/tap-workday-raas
+
+this-version: "1"
+
+api: |
+  [SOAP API](https://community.workday.com/api){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true
+
+api-type: "platform.workday-raas" 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Enterprise"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. There are no schema files for this integration.
```

Comment:
> There are no schema files for this integration.

This isn't needed - users also won't know what a `schema file` is. That's our internal term.

Also, what data does the integration replicate? How does Stitch get info about what data to replicate? You should put that info in here and point to the `Schema` section (see my comment there) where you can put additional info.